tableauserverclient/models/interval_item.py

```diff
@@ -0,0 +1,99 @@
+import xml.etree.ElementTree as ET
+from datetime import datetime, time
+from .. import NAMESPACE
+
+
+class IntervalItem(object):
+    class Frequency:
+        Hourly = "Hourly"
+        Daily = "Daily"
+        Weekly = "Weekly"
+        Monthly = "Monthly"
+
+    class Occurrence:
+        Hours = "Hours"
+        Minutes = "Minutes"
+        WeekDay = "weekDay"
+        MonthDay = "monthDay"
+
+    class Day:
+        Sunday = "Sunday"
+        Monday = "Monday"
+        Tuesday = "Tuesday"
+        Wednesday = "Wednesday"
+        Thursday = "Thursday"
+        Friday = "Friday"
+        Saturday = "Saturday"
+        LastDay = "LastDay"
+
+    def __init__(self, frequency, interval, start_time, end_time=None):
+        self.end_time = end_time
+        self.frequency = frequency
+        self.interval = interval
+        self.start_time = start_time
+
+    @staticmethod
+    def _validate_time(t):
```

Comment:
This might be useful just as a function at the module level, and returns true or false, then the classes that use it can decide what exception to throw, or to handle some other way (like using a default time)